ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-adaptive-virtual-topology.j2

```diff
@@ -28,6 +28,50 @@ Topology role: {{ router_adaptive_virtual_topology.topology_role }}
 {%         endif %}
 {%     endif %}
 
+{%     if router_adaptive_virtual_topology.profiles is arista.avd.defined
+        and router_adaptive_virtual_topology.profiles | length > 0 %}
+#### AVT Profiles
+
+| Profile name | Load balance policy | Internet exit policy |
+| ------------ | ------------------- | -------------------- |
+{%         for profile in router_adaptive_virtual_topology.profiles | arista.avd.natural_sort('name') %}
+| {{ profile.name }} | {{ profile.load_balance_policy | arista.avd.default("-") }} | {{ profile.internet_exit_policy | arista.avd.default("-") }} |
+{%         endfor %}
+{%     endif %}
+
+{%     if router_adaptive_virtual_topology.policies is arista.avd.defined
+        and router_adaptive_virtual_topology.policies | length > 0 %}
+#### AVT Policies
+{%         for policy in router_adaptive_virtual_topology.policies | arista.avd.natural_sort('name') %}
+
+##### AVT policy {{ policy.name }}
+| Application profile | AVT Profile | Traffic Class | DSCP |
+| ------------------- | ----------- | ------------- | ---- |
+{%             for matcher in policy.match_clauses %}
+| {{ matcher.application_profile }} | {{ matcher.avt_profile | arista.avd.default("-") }} | {{ matcher.traffic_class | arista.avd.default("-") }} | {{ matcher.dscp | arista.avd.default("-") }}
+{%             endfor %}
+{%         endfor %}
+{%     endif %}
+
+{%     if router_adaptive_virtual_topology.vrfs is arista.avd.defined
+        and router_adaptive_virtual_topology.vrfs | length > 0 %}
+#### VRFs configuration
+{%         for vrf in router_adaptive_virtual_topology.vrfs | arista.avd.natural_sort('name') %}
+
+##### VRF {{ vrf.name }}
+{%             if vrf.policy is arista.avd.defined %}
+| AVT policy |
+| ---------- |
+| {{ vrf.policy | arista.avd.default("-") }}
+{%             endif %}
+
+| AVT Profile | AVT ID |
+| ----------- | ------ |
+{%             for profile in vrf.profiles | arista.avd.natural_sort('id') %}
+| {{ profile.name }} | {{ profile.id }} |
+{%             endfor %}
+{%         endfor %}
```

Comment:
if statement around here as well in case there is no profile defined for the VRF